libraries/ms-helper/src/x86/ads1015.c

```diff
@@ -23,6 +23,9 @@
 #define ADS1015_CHANNEL_ARBITRARY_READING 0
 
 #ifdef MPXE
+#undef ADS1015_CHANNEL_UPDATE_PERIOD_US
```

Comment:
Is this macro defined elsewhere? Why do we need this undef?

Reply:
yes, it's defined 4 lines higher for normal stm32f0xx and x86 unit test use